### DISCUSSION TITLE
Lab Assignment #6

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/22S-ASIAAM-191A#33] [Assignments]
## Due 5/18 before class 

Please use this thread to post your [lab assignment's #6](https://albertkun.github.io/22S-ASIAAM-191A/assignments/week7/lab_assignment/) link, which is the GitHub pages URL. 

[Click here for a reminder on how to get it](https://albertkun.github.io/22S-ASIAAM-191A/labs/week1/3/#using-github-pages).

### For example

https://albertkun.github.io/22s-asiaam-191a-sample/week7/index.html

**Comment by daikis12** [comment]
https://daikis12.github.io/22s-191a-assignments/week7/index.html

**Comment by ananyakodali17** [comment]
https://ananyakodali17.github.io/asian-am-191a-labs/week7/

**Comment by bittertang** [comment]
https://bittertang.github.io/22s-asiaam-191a-labs/week%207/

**Comment by stefuntran** [comment]
https://stefuntran.github.io/22s-asiaam-191a-labs/week%207/index.html

**Comment by kris10fu** [comment]
https://kris10fu.github.io/22s-asiaam-191a-labs/week7/index.html

**Comment by emmashanson** [comment]
https://emmashanson.github.io/22s-asianam-191a-labs/week7/index.html

**Comment by XiongHer** [comment]
https://xiongher.github.io/22s-asiaam-191a-labs/week7/index.html

**Comment by ntran27** [comment]
https://ntran27.github.io/22s-asiaam-191a-labs/week7/index.html
@albertkun I tried to adjust the grid on my lab but can't seem to get it right. Can you advise?

**Reply by albertkun** [reply]
> https://ntran27.github.io/22s-asiaam-191a-labs/week7/index.html @albertkun I tried to adjust the grid on my lab but can't seem to get it right. Can you advise?

your `the_map` div is in the wrong place 
![image](https://user-images.githubusercontent.com/8574425/168960827-9c6c7a38-f63e-41a8-93be-73522220e3fc.png)

**Reply by ntran27** [reply]
thank you! I believe it's all fixed now.

**Reply by albertkun** [reply]
you're welcome

**Comment by kericsp** [comment]
https://kericsp.github.io/aa191-labs/week7/index.html

**Comment by hajovel** [comment]
https://hajovel.github.io/211S-191A-Assignments/week7/index.html

**Comment by kaittli** [comment]
https://kaittli.github.io/22s-asiaam-191a-labs/week7/index.html

**Comment by shubhgupta2018** [comment]
https://shubhgupta2018.github.io/22s-191a-assignments/week-7/index.html

**Comment by penelopelamb** [comment]
https://penelopelamb.github.io/22s-aas-191a-labs/week7/iceCream.html

**Comment by antonyblin** [comment]
https://antonyblin.github.io/aas191a-assignments/week7/index.html

**Comment by angelineeow** [comment]
https://angelineeow.github.io/22s-asiaam-191a-labs/week7/survey.html

**Comment by joonyloony** [comment]
https://joonyloony.github.io/22s-191a-assignments/week7/index.html

**Comment by spolcharoen** [comment]
https://spolcharoen.github.io/22s-asiaam-191a-labs/week7/index.html

**Comment by NishuKrish** [comment]
https://nishukrish.github.io/AsianAm191A-Nish/week7/index.html

**Comment by willa-21** [comment]
https://willakurland.github.io/aa191-labs/Week-7/index.html

**Comment by ashley-lu** [comment]
https://ashley-lu.github.io/s22-asiaam-191a-assignments/week7lab/survey.html

**Comment by codingdianat** [comment]
https://codingdianat.github.io/aa191-assignments/lab6/index.html
